### DISCUSSION TITLE
Where to find file docker-compose.openvino.yml

### DISCUSSION THREAD

**Discussion by mfoglio** [cvat-ai/cvat#3136] [General]
I'd like to use CVAT with semi-automatic labeling with Deep Extreme Cut.
I am following the instruction but it is not clear to me where to take the file `docker-compose.openvino.yml`.
Also, will this work on macOS Big Sur?

**Comment by nmanovic** [comment]
Hi,

Did you read the guide? https://github.com/openvinotoolkit/cvat/blob/develop/cvat/apps/documentation/installation_automatic_annotation.md

**Reply by ainureg** [reply]
> Hi,
> 
> Did you read the guide? https://github.com/openvinotoolkit/cvat/blob/develop/cvat/apps/documentation/installation_automatic_annotation.md

Hey, 

link is broken
I tried to follow this https://github.com/openvinotoolkit/cvat/tree/1257a1b245a3c9ec1a21c084b2db9ffd56954106/components/openvino, but still had no success to find any docker-compose.openvino.yml

btw I guess we (me and the author) both follow this
https://github.com/openvinotoolkit/cvat/blob/1257a1b245a3c9ec1a21c084b2db9ffd56954106/cvat/apps/dextr_segmentation/README.md

But there is rly no any newer documentation about dextr_segmentation
